lib/dfe/reference_data/degrees/institutions.rb

```diff
@@ -2376,17 +2388,20 @@ module Degrees
             ukprn: '10020381',
             'has_never_awarded_degrees' => true },
           '203f182c-1425-ec11-b6e6-000d3adf095a' =>
-          { name: 'UA92',
+          { name: 'University Academy 92',
             suggestion_synonyms: [],
-            match_synonyms: ['University Academy 92'],
+            match_synonyms: ['UA92'],
             hesa_itt_code: '434',
             dttp_id: '203f182c-1425-ec11-b6e6-000d3adf095a',
             ukprn: '10067648',
             'has_never_awarded_degrees' => true },
           'e53e182c-1425-ec11-b6e6-000d3adf095a' =>
-          { name: 'TEDI-London',
+          { name: 'The Engineering and Design Institute London (TEDI-London)',
```

Comment:
The "TEDI-London" brand seems slightly more prominent than the full name "The Engineering and Design Institute London" on [their website](https://tedi-london.ac.uk) so I wonder if it's worth reversing this order, on the assumption that users are more likely to type in the acronym?

```suggestion
          { name: 'TEDI-London (The Engineering and Design Institute London)',
```